README.md

```diff
@@ -185,7 +157,9 @@ After installing the plugin and syncing your Android Studio project with Gradle,
                     // folder with the name of the bundle passed in, e.g. index.android.bundle
                     .setJSBundleFile(codePush.getBundleUrl("index.android.bundle"))
                     
-                    // 6. Expose the CodePush module to JavaScript.
+                    // 6. DELETE THIS LINE --> .addPackage(new CodePushReactPackage())
```

Comment:
The version of the CodePush plugin that uses RNPM isn't going to support <RN 0.19.0 versions, so we presumably don't even need this section anymore right?